client/src/components/App.jsx

```diff
@@ -1,10 +1,94 @@
 import React, { Component } from 'react';
 
 class App extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      imgBoxesStatus: {
+        one: "inactive",
+        two: "inactive",
+        three: "inactive",
+        four: "inactive",
+        five: "inactive"
+      }
+    };
+
+    this.mouseOverHandler = this.mouseOverHandler.bind(this);
+    this.mouseOutHandler = this.mouseOutHandler.bind(this);
+  }
+
+  mouseOverHandler(event) {
+
+    let eventTargetName = event.target.getAttribute('name');
+
+    let tempObj = {};
+
+    // Image with mouseOver is active , rest all are blur
+    for (let currentKey in this.state.imgBoxesStatus) {
+      if (currentKey === eventTargetName) {
+        tempObj[currentKey] = 'active';
+      } else {
+        tempObj[currentKey] = 'blur';
+      }
+    }
+
+    this.setState({
+      imgBoxesStatus: tempObj
+    }
+      // , () => console.log(this.state.imgBoxesStatus)
+    );
+
+  }
+
+  mouseOutHandler() {
+
+    let tempObj = {};
```

Comment:
Again, would probably name this something other than tempObj. I see what it's doing, but you could make it more clear. Something that's kind of cool that you could look at is to manipulate the state directly by using a callback in setState:

example modified from https://reactjs.org/docs/state-and-lifecycle.html to apply to your example:

this.setState(function(state, props) {
     for (let currentKey in state.imgBoxesStatus) {
          state.imgBoxesStatus[currentKey] = 'inactive';
    }
  return state
});

Reply:
I will change the variable names, but for code readability for future me, I would like to use seperate  object variable. Thanks for the suggestion.